src/gnmi_clients/src/github.com/openconfig/gnmi/cli/cli.go

```diff
@@ -0,0 +1,389 @@
+/*
+Copyright 2017 Google Inc.
```

Comment:
Why are we committing these files? Why can't we pull at runtime with a fixed commit-id ?

Reply:
I have removed the entire gnmi_client directory now, and instead "go get" them and then patch as needed.